Assets/AnkrSDK/Scripts/Core/Implementation/ContractEventSubscriber.cs

```diff
@@ -87,6 +87,18 @@ private async UniTaskVoid Update()
 
 			return eventSubscription;
 		}
+		
+		public async UniTask<IContractEventSubscription> Subscribe<TEventType>(EventFilterRequest<TEventType> evFilter, string contractAddress,
```

Comment:
I don't think that method overload is a good idea here, maybe we will name it as 
SubscribeWithFilterRequest / SubscribeWithTopicFilters ?